Alignment/OfflineValidation/bin/Zmumumerge.cc

```diff
@@ -0,0 +1,310 @@
+#include <iostream>
+#include <TString.h>
+#include <TCanvas.h>
+#include "TBranch.h"
+#include "TTree.h"
+#include "TSystem.h"
+#include "TH1F.h"
+#include "TH1D.h"
+#include "TF1.h"
+#include "TMath.h"
+#include "TLegend.h"
+#include "TLatex.h"
+#include <TH2F.h>
+#include <TCut.h>
+#include <TStyle.h>
+
+#include <TChain.h>
+#include <TFile.h>
+#include <cmath>
+#include <string>
+
+#include "TH1.h"
+#include "TROOT.h"
+#include "TPad.h"
+#include "TRandom.h"
+#include "THStack.h"
+#include "TH2.h"
+#include "TLatex.h"
+#include "TPaveText.h"
+#include "TFrame.h"
+#include "TAxis.h"
+#include <iostream>
+#include <fstream>
+#include <iomanip>
+#include <TCut.h>
+
+#include <RooFit.h>
+#include <RooRealVar.h>
+#include <RooDataHist.h>
+#include <RooDataSet.h>
+#include <RooHistPdf.h>
+#include <RooPlot.h>
+#include <RooAddPdf.h>
+//#include <RooGlobalFunc.h>
+#include <RooGaussian.h>
+#include <RooPolynomial.h>
+#include <RooGenericPdf.h>
+#include <RooFormulaVar.h>
+#include <RooChebychev.h>
+#include <RooExponential.h>
+#include <RooFitResult.h>
+#include <RooWorkspace.h>
+#include <RooMCStudy.h>
+#include "RooHist.h"
+#include "RooConstVar.h"
+#include "RooMsgService.h"
+#include "RooVoigtian.h"
+#include "FitWithRooFit.cc"
+#include "toolbox.h"
+#include "Options.h"
+#include <boost/filesystem.hpp>
+#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/json_parser.hpp>
+#include <boost/optional.hpp>

```

Comment:
I don't see anywhere where you are using `boost::optional`: why do you load the header? 

This comment may apply to RooFit headers are well -- please check.

Reply:
I have deleted it.

> in the last push I've taken the latest version from https://github.com/gu180/TrackderAlignment_DiMuonValidation/tree/all-in-one.
> I am not sure all the comments have been addressed.
> Unfortunately since apparently @gu180 is using dos formatting while everybody else is using unix, the comparison gets spoiled as the entire file looks changed.

I have renewed the DiMuonValidation.cc file. Is there still dos and unix problem?